csharp_webapi/11.QnAMaker/README.md

```diff
@@ -0,0 +1,50 @@
+﻿This sample shows how to integrate QnA Maker in a simple bot with ASP.Net Web API and Application Insights. 
+
+# Concepts introduced in this sample
+The [QnA Maker Service](https://www.qnamaker.ai) enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes.
+
+The [Application Insights](https://azure.microsoft.com/en-us/services/application-insights/) enables you to discover actionable insights through application performance management and instant analytics.
+
+In this sample, we demonstrate how to use the QnA Maker service to answer questions based on a FAQ text file as input.
+
+# To try this sample
+
+- Clone the samples repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+
+## Prerequisites
+- Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/set-up-qnamaker-service-azure) to create a QnA Maker service.
+- Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/migrate-knowledge-base) to import the [sample.qna](sample.qna) to your newly created QnA Maker service.
+- Update [QnABot.bot](QnABot.bot) with your kbid (KnowledgeBase Id) and endpointKey in the "qna" services section. You can find this information under "Settings" tab for your QnA Maker Knowledge Base at [QnAMaker.ai](https://www.qnamaker.ai)
```

Comment:
We should document the CLI tools workflow since that makes developers life easier.

```bash
> npm i -g msbot qnamaker

> qnamaker create kb --in <kb.json> --subscriptionKey <Key> --msbot | msbot connect qna --stdin
```